recipes/vcf-validator/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "vcf-validator" %}
+{% set version = "0.9.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/EBIvariation/vcf-validator/releases/download/v{{ version }}/vcf_validator_linux        # [linux]
```

Comment:
Is it possible to compile the packages here?

Reply:
I've been [trying](https://github.com/tcezard/bioconda-recipes/blob/vcf-validator-src/recipes/vcf-validator/meta.yaml) unsuccessfully.

We're planning on removing some of [our dependencies](https://github.com/EBIvariation/vcf-validator/issues/229) in the future which should help simplify the build process. But in the meantime there is [some demand](https://github.com/EBIvariation/vcf-validator/issues/219) to have the vcf-validator in bioconda